images/build/Dockerfiles/Dockerfile

```diff
@@ -19,16 +19,16 @@ RUN apt-get update \
     && apt-get install -y --no-install-recommends \
         # Required for .NET Core 1.1
         libunwind8 \
-    && rm -rf /var/lib/apt/lists/*
-
+    && rm -rf /var/lib/apt/lists/* \
```

Comment:
How many layers does the full build image have after this change?

Reply:
60